shared/naturalcrit/codeEditor/fold-css.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+	registerHomebreweryHelper : function(CodeMirror) {
+		CodeMirror.registerHelper('fold', 'homebrewerycss', function(cm, start) {
+
+			// BRACE FOLDING
+			const startMatcher = /\{[ \t]*$/;
+			const endMatcher = /\}[ \t]*$/;
+			const prevLine = cm.getLine(start.line);
+
+
+			if(prevLine.match(startMatcher)) {
+				const lastLineNo = cm.lastLine();
+				let end = start.line + 1;
+				let braceCount = 1;
+
+				while (end < lastLineNo) {
+					const curLine = cm.getLine(end);
+					if(curLine.match(startMatcher)) braceCount++;
+					if(curLine.match(endMatcher)) braceCount--;
+					if(braceCount == 0) break;
+					++end;
+				}
+
+				return {
+					from : CodeMirror.Pos(start.line, 0),
+					to   : CodeMirror.Pos(end, cm.getLine(end).length)
+				};
+			}
+
+			// IMPORT FOLDING
+
+			const importMatcher = /^@import.*?[;]/;
```

Comment:
I know I wrote this, but after further testing, I believe the square brackets in this regex are unnecessary. That said, it still works fine as is.

Reply:
I know I see these without the training semicolon frequently.

Shouldn't this be `/^@import.*/` ?

Reply:
At that point, it could just be `^@import`... I was originally working towards the semicolon as a mandatory terminator; every example at https://developer.mozilla.org/en-US/docs/Web/CSS/@import terminates with one. It's far less of an issue if `@import` is always a single line.

Reply:
That does seem to be the case

Reply:
Let's remove the square brackets because they do nothing.

```suggestion
			const importMatcher = /^@import.*?;/;
```

Looks like you need the semicolon though or anything after is not parsed.